common/test/acceptance/pages/studio/library.py

```diff
@@ -168,6 +169,7 @@ def count(self, count):
         Sets value of children count input
         """
         count_text = self.get_setting_element(self.COUNT_LABEL)
+        count_text.send_keys(Keys.BACK_SPACE)
```

Comment:
could you explain why this step is necessary? I assume it will work on firefox, too?